moped-editor/src/views/projects/projectView/ProjectComponents/utils/makeFeatureCollections.js

```diff
@@ -13,57 +13,33 @@ export const makeFeatureFromProjectGeographyRecord = (record) => ({
   geometry: record.geometry,
 });
 
-/**
- * Create an object map of component feature collections by component id
- * Ex. { 1: { type: "FeatureCollection", features: [...] } }
- * @param {Object} data - data returned by GET_PROJECT_COMPONENTS query
- * @returns {Object} - map of component feature collections by component id
- */
-export const useComponentFeatureCollectionsMap = (data) =>
-  useMemo(() => {
-    if (!data?.project_geography) return {};
-
-    const componentGeographyMap = {};
-
-    data.project_geography.forEach((component) => {
-      const currentComponentId = component.component_id;
-      const currentFeature = makeFeatureFromProjectGeographyRecord(component);
-
-      if (!componentGeographyMap[currentComponentId]) {
-        componentGeographyMap[currentComponentId] = {
-          type: "FeatureCollection",
-          features: [currentFeature],
-        };
-      } else {
-        componentGeographyMap[currentComponentId] = {
-          type: "FeatureCollection",
-          features: [
-            ...componentGeographyMap[currentComponentId].features,
-            currentFeature,
-          ],
-        };
-      }
-    });
-
-    return componentGeographyMap;
-  }, [data]);
-
```

Comment:
Looks like this function is a holdover from an earlier iteration of the map...

Reply:
Thanks for removing this. Yep, this is a leftover from before related components. 🙏